### PR TITLE
Robotv1

### PR DESCRIPTION
**Pull request by alecLiu14**
Compare what's been added for limelight v1

### CODE REVIEW

File: src/main/java/frc/robot/Constants.java [hunk 1]
```diff
@@ -67,11 +67,11 @@
...
-    public static final double STAGE_1_SPEED = 0.8;
-    public static final double STAGE_2_SPEED = 0.5;
-    public static final double STAGE_3_SPEED = 0.3;
-    public static final double STAGE_4_SPEED = 0.1;
-    public static final double STAGE_5_SPEED = 0.05;
+    public static final double STAGE_1_SPEED = 1.0;
+    public static final double STAGE_2_SPEED = 0.8;
+    public static final double STAGE_3_SPEED = 0.5;
+    public static final double STAGE_4_SPEED = 0.3;
+    public static final double STAGE_5_SPEED = 0.1;
```
Comment (alecLiu14) [inline src/main/java/frc/robot/Constants.java:75]: changes how fast limelight moves the robot, low priority but worth testing new numbers if there's enough time.

File: src/main/java/frc/robot/DrivetrainConstants.java [hunk 2]
```diff
@@ -32,20 +32,20 @@
...
     public static final int BACK_LEFT_MODULE_STEER_MOTOR = 8; 
     public static final int BACK_LEFT_MODULE_STEER_ENCODER = 12; 
-    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(131.3); // FIXME Measure and set back left steer offset 136 134
+    public static final double BACK_LEFT_MODULE_STEER_OFFSET = -Math.toRadians(134); // FIXME Measure and set back left steer offset 136 134
                                                                                 //313
     public static final int BACK_RIGHT_MODULE_DRIVE_MOTOR = 5; 
     public static final int BACK_RIGHT_MODULE_STEER_MOTOR = 6; 
     public static final int BACK_RIGHT_MODULE_STEER_ENCODER = 13; 
-    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(190.7); // FIXME Measure and set back right steer offset 37 340
+    public static final double BACK_RIGHT_MODULE_STEER_OFFSET = -Math.toRadians(340.0); // FIXME Measure and set back right steer offset 37 340
```
Comment (alecLiu14) [inline src/main/java/frc/robot/DrivetrainConstants.java:50]: don't copy these for v2, these numbers are only for v1 swervedrive

File: src/main/java/frc/robot/FinalContainer.java [hunk 3]
```diff
@@ -73,14 +80,22 @@ private void configureButtonBindings() {
...
     buttonLeftJoystickButton.whenPressed(new AutoMeter(m_drivetrainSubsystem, m_CatapultLeftSubsystem, m_CatapultLeftSubsystem, m_IntakeMotors));
     buttonRightJoystickButton.whenPressed(new AutoCommand(m_drivetrainSubsystem));
+
+
+    // limelight commands
+    buttonB.whenPressed(new GoToDistanceLimelight(70, m_drivetrainSubsystem, ll));
+    buttonA.whenPressed(new SwitchLimelightStream());
+    buttonX.whenPressed(new TurnToZeroLimelight(0, m_drivetrainSubsystem, ll));
+    buttonY.whenPressed(new AlignAndMoveToLimelight(60, m_drivetrainSubsystem, ll));
+
```
Comment (alecLiu14) [inline src/main/java/frc/robot/FinalContainer.java:98]: worth going through and setting which buttons do which, imported the necessary commands for limelight and stole some buttons from other commands in order to test limelight

File: src/main/java/frc/robot/commands/AutoAlignCommands/SwitchLimelightStream.java [hunk 4]
```diff
@@ -0,0 +1,34 @@
+// Copyright (c) FIRST and other WPILib contributors.
```
Comment (alecLiu14) [inline src/main/java/frc/robot/commands/AutoAlignCommands/SwitchLimelightStream.java:1]: maybe keep? if we want to keep USB camera as the "main" one then we don't need it

File: src/main/java/frc/robot/commands/AutoAlignCommands/TurnToZeroLimelight.java [hunk 5]
```diff
@@ -51,12 +53,13 @@ public TurnToZeroLimelight(double targetAngleDegrees, DrivetrainSubsystem drivet
...
     // ll.printLoc();
     // Shuffleboard.getTab("limelight-test").add("test");
-    System.out.println("***********" + ll.getDistance() + "**********");
+    // System.out.println("***********" + ll.getDistance() + "**********");
   }
 
 
   @Override
   public void execute() {
+    super.execute();
```
Comment (alecLiu14) [inline src/main/java/frc/robot/commands/AutoAlignCommands/TurnToZeroLimelight.java:62]: fixed the turn, keep all changes here

File: src/main/java/frc/robot/subsystems/LimelightPortal.java [hunk 6]
```diff
@@ -17,9 +17,10 @@
    * Creates a new LimelightPortal.
    */
   public LimelightPortal() {
-     shooterLL = NetworkTableInstance.getDefault().getTable("limelight-shooter"); 
-     shooterLL.getEntry("stream").setNumber(0);
+    shooterLL = NetworkTableInstance.getDefault().getTable("limelight-shooter");
```
Comment (alecLiu14) [inline src/main/java/frc/robot/subsystems/LimelightPortal.java:20]: attempted to set the stream mode, didn't work but keep the code nonetheless, review what we do need after stream mode problem is fixed

**Review by alecLiu14 — COMMENTED** [review]
all changes are all around limelight, there might be some parts conflicting each other (like the final container buttons setting)

**Comment by RobertTheBlair** [comment]
replaced by better PR

**Pull request closed without merging**